build.sh

```diff
@@ -92,7 +92,7 @@ check_ret() {
 set_downloader() {
     test_program_installed wget
     if [[ $? -ne 0 ]] ; then
-        DOWNLOADER=wget
+        DOWNLOADER="wget -nd"
```

Comment:
Why do you need ``--no-directories``?

Reply:
Without it the binary ends up in the wrong place. Maybe because of different defaults on Unix or because of a ~/.wgetrc?